src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -51,9 +52,10 @@ ClusterResourceScheduler::ClusterResourceScheduler(
         cluster_resource_manager_->AddOrUpdateNode(local_node_id_, local_resource_update);
       });
   cluster_resource_manager_->AddOrUpdateNode(local_node_id_, local_node_resources);
-  scheduling_policy_ = std::make_unique<raylet_scheduling_policy::SchedulingPolicy>(
-      local_node_id_, cluster_resource_manager_->GetResourceView(),
-      [this](auto node_id) { return this->NodeAlive(node_id); });
+  scheduling_policy_ =
+      std::make_unique<raylet_scheduling_policy::CompositeSchedulingPolicy>(
+          local_node_id_, cluster_resource_manager_->GetResourceView(),
+          [this](auto node_id) { return this->NodeAlive(node_id); });
```

Comment:
The lambda will be invoked in a high frequency, it is better to use `const auto &node_id` to avoid the copy of `node_id`.